Documentation/Helm-Charts/operator-chart.md

```diff
@@ -54,6 +54,7 @@ The following table lists the configurable parameters of the rook-operator chart
 | `crds.enabled` | Whether the helm chart should create and update the CRDs. If false, the CRDs must be managed independently with deploy/examples/crds.yaml. **WARNING** Only set during first deployment. If later disabled the cluster may be DESTROYED. If the CRDs are deleted in this case, see [the disaster recovery guide](https://rook.io/docs/rook/latest/Troubleshooting/disaster-recovery/#restoring-crds-after-deletion) to restore them. | `true` |
```

Comment:
Add something like `***` next to the options and add single notes at the bottom of the table ? 

csi-addons operation like reclaimspace and future keyrotation will not be supported too if this option is enabled since we'll have no VolumeAttachments to determine which node the pvc is mounted on.

Reply:
yes that true but we need to find an alternative way to support it.